design-toolkit/components/src/components/input/input.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Input } from './';
+
+const meta: Meta<typeof Input> = {
+  title: 'Components/Input',
+  component: Input,
+  args: {
+    disabled: false,
```

Comment:
```suggestion
    disabled: false,
    isInvalid: false
```
This sets a default in the story that makes the control a little more intuitive.

<img width="200" alt="CleanShot 2025-07-23 at 13 58 14" src="https://github.com/user-attachments/assets/db9db192-f0b8-4924-892d-7e2f1bc42d11" />